Examples/UIExplorer/js/TextExample.ios.js

```diff
@@ -22,6 +22,9 @@
  */
 'use strict';
 
+const Platform = require('Platform');
+const isTVOS = (Platform && Platform.OS === 'ios' && Platform.SystemName === 'tvOS');
```

Comment:
Sure I can do that.  I think I started doing a null check on Platform when an earlier version of my changes caused the TouchableHighlight JS unit test to need that check :)